include/fmt/printf.h

```diff
@@ -306,12 +304,6 @@ class printf_arg_formatter
     write_null_pointer(char_type());
     return this->out();
   }
-
-  /** Formats an argument of a custom (user-defined) type. */
-  iterator operator()(typename basic_format_arg<context_type>::handle handle) {
```

Comment:
This should remain here. Please see `arg_formatter` for an example.

Reply:
See comment at printf_arg_formatter_proxy